pages/common/javac.md

```diff
@@ -5,3 +5,19 @@
 - Compile a .java file:
 
 `javac {{filename.java}}`
+
+- Compile two or more .java files:
+
+`javac {{filename1.java}} {{filename2.java}} {{filename3.java}}`
+
+- Compile all .java files in current directory:
+
+`javac {{*.java}}`
+
+- Compile a .java file and place class file in a specific directory:
+
+`javac -d {{path/to/some/directory}} {{filename.java}}`
+
+- Compile two or more .java files and place class files in a specific directory:
```

Comment:
I agree with your suggestions, I'll make the changes tonight when I'm in front of a computer. Do you think a command line beginner would be able to combine the previous examples to come up with the last one?

-- Muhammad-Sharif Moustafa

> On Jan 15, 2016, at 3:12 AM, Waldir Pimenta <notifications@github.com> wrote:
> 
> In pages/common/javac.md:
> 
> > @@ -5,3 +5,19 @@
> >  - Compile a .java file:
> >  
> >  `javac {{filename.java}}`
> > +
> > +- Compile two or more .java files:
> > +
> > +`javac {{filename1.java}} {{filename2.java}} {{filename3.java}}`
> > +
> > +- Compile all .java files in current directory:
> > +
> > +`javac {{*.java}}`
> > +
> > +- Compile a .java file and place class file in a specific directory:
> > +
> > +`javac -d {{path/to/some/directory}} {{filename.java}}`
> > +
> > +- Compile two or more .java files and place class files in a specific directory:
> This one is unnecessary IMO, but if you prefer to keep it, use "several" here too.
> 
> —
> Reply to this email directly or view it on GitHub.
>